### ISSUE TITLE
Continued support for mid-scale radio instruments as transient discovery machines

### ISSUE THREAD

**Issue by cwjames1983** [ethrane/transients#17]
In the last decade, both ASKAP and MWA have proven themselves world-leading instruments for transient discovery, due to their wide fields of view, and technical search capabilities. This has been led by dedicated transient search modes with ASKAP (the CRAFT Collaboration), but also new methods for searching for transients in images (processing of GLEAM on MWA, VAST on ASKAP). These wide FOV capabilities also enable follow-up observations of gravitational waves and GRBs. Facilities such as ATCA also play a key role in follow-up observations of transients (e.g. the binary neutron star merger) and the unravelling of their astrophysics.

In the next decade, the opportunity will arise for further technical upgrades to these instruments (e.g. more sensitive PAFs on ASKAP), and/or a new mid-scale radio instrument (and all-sky monitoring telescope) which is dedicated to transient discovery. We should continue to support such projects, which will likely cost in the $1-$10M bracket, and keep Australia at the forefront of transient discovery and follow-up.

[Updated to mention ATCA as per Gemma's and Adelle's comments, though happy to change text details!]

**Comment by cwjames1983** [comment]
@ethrane How do we add the "infrastructure" label to this?

**Comment by adellej** [comment]
I think it might also be important to mention ATCA here, given it's unique frequency coverage range in the southern hemisphere essential for many synchrotron transients such as GRBS and TDEs.

**Comment by gemmaanderson** [comment]
I second Adelle - widefield Australian radio telescopes such as MWA and ASKAP are great transient discovery machines. However, we still need small scale facilities with high time resolution and broadband frequency coverage to follow-up them up and properly explore the science.

**Comment by nhurleywalker** [comment]
Could we rephrase this as: "Support for the SKA precursors and technology development for new small-scale facilities"?

**Comment by cwjames1983** [comment]
@nhurleywalker I think your title works too - I wanted in mine to emphasise the transient discovery aspect, but "continued support" isn't very headline-grabbing (also, Starry turned off my computer halfway through typing this).

Question: do we combine this with SKA? I would prefer *not* to, as I do think precursors should be emphasised separately

**Comment by nhurleywalker** [comment]
@cwjames1983 Yeah, I took the transient-speific aspect out because I _thought_ the different SWG infrastructure priorities were all getting summarised at a higher level, and supporting the SKA precursors cuts across most (all?) science groupings. But I'm a little hazy on how the next stage works.

**Comment by ddobie** [comment]
@adellej the ATCA is also unique in that it's the only cm-wavelength facility capable of automated rapid-response observations, and even regular ToOs generally have a faster turnaround than the VLA. There's probably a lot of good science in rapid follow-up of Rubin discoveries which it's geographically suited to as well!

I think we should really push for making sure the ATCA is mentioned somewhere - there's a very real possibility it gets shut down in the next decade as the SKA comes online.

**Comment by nhurleywalker** [comment]
Agreed, I don't want to minimise ATCA and Parkes here with my suggest rephrase. It's just "mid-scale" doesn't clearly capture for me the gist of what we want here.

**Comment by RonEkers** [comment]
Where Australia provides unique international access to Southern Hemisphere instruments, eg ATCA and Parkes, the decadal review should note this as an important return for the open access we have to many other unique international facilities.

**Comment by taramurphy** [comment]
> @adellej the ATCA is also unique in that it's the only cm-wavelength facility capable of automated rapid-response observations, and even regular ToOs generally have a faster turnaround than the VLA. There's probably a lot of good science in rapid follow-up of Rubin discoveries which it's geographically suited to as well!

The rapid response comment is only true assuming no upgrades. What if ASKAP was upgraded to have a rapid response capability?

**Comment by taramurphy** [comment]
Just an overall comment to keep in mind (and from the perspective of someone who hasn't attended the various discussion meetings so far). Remember a key purpose of the decadal plan is to priortise. What I'm seeing as I read these threads is our priorities are:
* The SKA - which from the governments perspective is the "best" radio telescope we have ever had by orders of magnitude. But yet will still need significantly more investment to roll out future phases.
* Keeping ASKAP, MWA, ATCA and upgrading them 
* A new transient monitoring telescope that we've only just heard about in this discussion 

From an outside perspective that's going to sound a lot like we want the big new SKA telescope, all the old telescopes, and also a completely new telescope in addition.

Talking to Elaine, I realise the purpose of these working groups is to generate ideas for the science case to be made in the decadal plan. So I think the most useful thing would be to try and articulate what science questions (a) are important and (b) can't be answered by the existing facilities if you want to make the case for new facilities.

I think that case needs to be articulated more clearly that Telescope Y will detect lots of [magnetars|pulsars|ulplos etc] but Telescope Y++ will detect even more of them :)

**Comment by Tonytravouillon** [comment]
I would generalise by removing the word "radio". We should support all wavelengths when it comes to mid-scales intruments.

**Comment by strocode** [comment]
I wanted to add a +1 for Parkes/Murriyang - the ultrawideband low is used extensively for follow-up of FRB and ultra-long period sources / weird stuff in the galaxy.
